source/draw/RasterBgr.ooc

```diff
@@ -62,13 +78,8 @@ RasterBgr: class extends RasterPacked {
 		(convert as Closure) free()
 	}
 	resizeTo: override func (size: IntVector2D) -> This {
-		result: This
-		if (this size == size)
-			result = this copy()
-		else {
-			result = This new(size)
-			RasterCanvas resizeNearestNeighbour(this buffer pointer as ColorBgr*, result buffer pointer as ColorBgr*, IntBox2D new(this size), IntBox2D new(result size), this stride, result stride, this bytesPerPixel)
-		}
+		result := This new(size)
+		result canvas draw(this, IntBox2D new(this size), IntBox2D new(size))
 		result
```

Comment:
In the future this function will be moved directly to `Image` class. For now not every Canvas subclass can draw images.